webapp/dojo_news/src/components/NewsListItemInput.vue

```diff
@@ -0,0 +1,24 @@
+<template>
+  <form @submit.prevent="$emit('create', title)">
+    <input type="text" v-model="title"/>
+    <input type="submit" value="Add"/>
+  </form>
+</template>
+
+<script>
+export default {
+  name: "NewsListItemInput",
+  emits: ["create"],
+  data() {
+    return {
+      title: ""
```

Comment:
When you create a new `NewsListItem` you should clear `title`. When I create new items I can see the `NewsListItemInput` has the previous input.

Reply:
Maybe we just like to make a spammer's life easier :smile: 
Thanks, Robert!